src/main/java/com/diffmin/App.java

```diff
@@ -129,6 +128,19 @@ public String displayModifiedModel(CtModel model) {
         return printer.prettyprint(cu);
     }
 
+    /** Computes the index at which the `insertedNode` has to be inserted. */
+    public int getInsertIndex(CtElement insertedNode) {
```

Comment:
As a general rule of thumb, keep visibility of methods/constructors/fields as limited as possible. This is definitely an implementation detail.

```suggestion
    private int getInsertIndex(CtElement insertedNode) {
```

Reply:
Oh, right. You pointed this out before. I will change the modifier. 😅